lib/glific/flows/contact_action.ex

```diff
@@ -92,7 +92,7 @@ defmodule Glific.Flows.ContactAction do
 
     with {false, context} <- has_loops?(context, body, messages) do
       attrs = %{
-        body: body,
+        body: MessageVarParser.parse(body, message_vars),
```

Comment:
@mohitgusain are you sure this is the fix ? Because I think we are changing the body in the create_and_send_message function also.

Please confirm and explain that and then we can merge.

Reply:
yes @pankaj-ag I am sure about the fix. While using the inactive message from the sheet, the body was not saved into the DB table in the correct format. In the body, the variable was getting saved into DB. Attaching a screenshot for reference.

<img width="919" alt="image" src="https://user-images.githubusercontent.com/30264862/231682452-0923f28b-bc7a-40a8-b7b9-b7449197850e.png">

Reply:
Okay let's chat about this